### ISSUE TITLE
Table headers not centering

### ISSUE THREAD

**Issue by astroDimitrios** [carpentries/varnish#161]
I'm not sure if anyone else is having this problem but I can't centre a table heading because of some bootstrap css which I think we may need to overwrite:

```css
th {
  text-align: revert;
}
```

Before:

![image](https://github.com/user-attachments/assets/949cc826-b979-4907-9d5a-430b9876d136)

After:

![image](https://github.com/user-attachments/assets/f157d08e-e176-4ac5-807d-1b156bc3bb18)

**Comment by froggleston** [comment]
Hey @astroDimitrios - whilst we can do this across all lessons, I'm not sure it works for some of the tables to aid readability, e.g.

<img width="799" height="432" alt="Image" src="https://github.com/user-attachments/assets/315d896f-f22d-4c23-8b65-1eff57b10e2b" />

In this example, I kind of prefer the existing left align?

<img width="784" height="438" alt="Image" src="https://github.com/user-attachments/assets/933587a2-8c5b-4918-90ce-1a98f7a5741b" />

**Comment by astroDimitrios** [comment]
@froggleston Yes I agree it doesn't work for all tables, I'm also not sure a solution to allow both options would be pretty. The [workbench component guide](https://carpentries.github.io/sandpaper-docs/episodes.html#tables) has a lot of info on Tables, and alignment of columns. Could we add a sentence something along the lines of:

Please note it is not possible to centre or right-align Table headers with the Workbench styling. (Or something phrased better)

**Comment by froggleston** [comment]
Sure, that would be a good addition. I can add something to that effect in the docs!
